### PR TITLE
For #8154 - XCUITests Fix or Disalbe intermittent failing tests

### PR DESCRIPTION
**Pull request by isabelrios**
Trying to fix some intermittents on Bitrise. Disable Unit Test

### CODE REVIEW

**Comment by isabelrios** [comment]
This PR will try to fix these issues:
#8190
#8189
#8178
#7867

File: XCUITests/Fennec_Enterprise_XCUITests.xctestplan [hunk 1]
```diff
@@ -77,7 +77,6 @@
         "NewTabSettingsTest\/testChangeNewTabSettingsShowYourHistory()",
         "NewTabSettingsTest\/testCheckNewTabSettingsByDefault()",
         "NewTabSettingsTest\/testToggleOffOnAdditionalContentBookmarks()",
-        "NoImageTests",
```
Comment (isabelrios) [inline XCUITests/Fennec_Enterprise_XCUITests.xctestplan:80]: I have to set it back as it was... updating PR. Same with the below test suite

**Review by SimonBasca — APPROVED** [review]

**Comment by SimonBasca** [comment]
@isabelrios should we close the issue from this PR?
https://github.com/mozilla-mobile/firefox-ios/issues/8190
https://github.com/mozilla-mobile/firefox-ios/issues/8189
https://github.com/mozilla-mobile/firefox-ios/issues/8178
https://github.com/mozilla-mobile/firefox-ios/issues/7867
https://github.com/mozilla-mobile/firefox-ios/issues/8119

**Comment by isabelrios** [comment]
Yes! @SimonBasca we can do that

**Pull request merged**